modules/upi-removing-hosts-from-a-upi-cluster.adoc

```diff
@@ -43,11 +43,11 @@ node/app1 drained
 
 . Delete the `customDeploy` specification from the `BareMetalHost` CR.
 
-.. Edit the `BareMetalHost` CR for the host by running the following command:
+. Delete the host by running the following command:
 +
 [source,terminal]
 ----
-$ oc edit bmh -n openshift-machine-api <host_name>
+$ oc delete bmh -n openshift-machine-api <bmh_name>
 ----
```

Comment:
Can a note be added here that this step can also be executed without step 2, but to note that the command will take time to complete as it takes time for the bmh state to change from deprovisioning to available.